index.bs

```diff
@@ -545,6 +545,16 @@ Authors of external specifications are encouraged to to add new modules rather t
 existing ones. Where it is desired to extend an existing module, it is preferred to integrate the
 extension directly into the specification containing the original module definition.
 
+### External specifications ### {#external-specifications}
+
+<em>This section is non-normative.</em>
+
+The following external specifications define additional WebDriver BiDi modules:
+
+1. <a href="https://www.w3.org/TR/permissions/">Permissions</a>
+
+1. <a href="https://webbluetoothcg.github.io/web-bluetooth/">Web Bluetooth</a>
```

Comment:
I also wanted to mention [WebAuthn](https://www.w3.org/TR/webauthn-2/#sctn-automation) but they only support Webdriver classic at the moment. It might be good to get in contact with them and ask for an addition for BiDi.